src/Tasks/Microsoft.NET.Build.Tasks/ResolveRuntimePackAssets.cs

```diff
@@ -99,9 +101,56 @@ void AddAsset(string assetPath, string assetType)
                 {
                     AddAsset(asset, "native");
                 }
+
+                runtimePackAssets.AddRange(EnumerateResourceAssets(runtimePackRoot, runtimeIdentifier, runtimePack));
             }
 
             RuntimePackAssets = runtimePackAssets.ToArray();
         }
+
+        private IEnumerable<ITaskItem> EnumerateResourceAssets(string runtimePackRoot, string runtimeIdentifier, ITaskItem runtimePack)
+        {
+            //  These hard-coded paths are temporary until we have "real" runtime packs, which will likely have a flattened structure
+            var directory = Path.Combine(runtimePackRoot, "runtimes", runtimeIdentifier, "lib", "netcoreapp3.0");
+            if (!Directory.Exists(directory))
+            {
+                yield break;
+            }
+
+            foreach (var subdir in Directory.EnumerateDirectories(directory))
+            {
+                foreach (var asset in EnumerateCultureAssets(subdir, runtimeIdentifier, runtimePack))
+                {
+                    yield return asset;
+                }
+            }
+        }
+
+        private IEnumerable<ITaskItem> EnumerateCultureAssets(string cultureDirectory, string runtimeIdentifier, ITaskItem runtimePack)
+        {
+            var culture = Path.GetFileName(cultureDirectory);
+
+            if (this.SatelliteResourceLanguages.Length > 1 &&
+                !this.SatelliteResourceLanguages.Any(lang => string.Equals(lang.ItemSpec, culture, StringComparison.OrdinalIgnoreCase)))
+            {
+                yield break;
+            }
+
+            foreach (var file in Directory.EnumerateFiles(cultureDirectory))
```

Comment:
Is it worth filtering to *.resources.dll here? We got burned by over-aggressively picking up random files fro runtime pack before. I realize the manifests should be the real solution here, but I wonder if we should more conservative in the meantime.

Reply:
I can add that.